src/clusterfuzz/_internal/bot/tasks/utasks/analyze_task.py

```diff
@@ -246,8 +244,8 @@ def update_testcase_after_crash(testcase, state, job_type, http_flag):
         bool(testcase.gestures))
 
 
-def execute_task(testcase_id, job_type):
-  """Runs analyze task."""
+def utask_preprocess(testcase_id, job_type, uworker_env):
```

Comment:
is `utask_preprocess` called anywhere? or is this just establishing a function to be used in a future PR?

Reply:
It is called in utasks/__init__